feat: Adds logging for every request to ksqlDB (#6615)
confluentinc/ksql: docs/operate-and-deploy/installation/server-config/config-reference.md

Toggles whether or not the processing log should include rows in log
messages. By default, this property has the value `false`.

### ksql.logging.skipped.response.codes

> A comma-separated list of HTTP response codes to skip when doing

Reviewer: ```suggestion
A comma-separated list of HTTP response codes to skip during server
```

> This is a comma separated list of http response codes to skip when doing

Reviewer 2: ```suggestion
A comma-separated list of HTTP response codes to skip when doing
```

request logging. This is useful for ignoring certain 4XX errors that you
might not want to show up in the logs.

### ksql.logging.rate.limited.request.paths

> This is a list of path:rate_limit pairs, to rate limit the server request

Reviewer 2: ```suggestion
A list of `path:rate_limit` pairs, to limit the rate of server request
```

logging.  This is useful for requests that are coming in at a high rate,

> such as for pull queries. It makes it possible to see the logs at a low rate

Reviewer 2: ```suggestion
such as for pull queries. This setting enables seeing the logs when the request rate is low,
```

> of requests, but start dropping them when they go over the threshold.

Reviewer 2: ```suggestion
and dropping them when they go over the threshold.
```
